src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -7,7 +7,7 @@
         },
         "DHCP_SERVER": {
             "192.0.0.8": {},
-            "192.0.0.8": {}
```

Comment:
Do not delete old test data. I believe it is intentionally duplicating above item, and it is a corner case to test.

Reply:
This is added in sonic-dhcp_server.yang PR: https://github.com/sonic-net/sonic-buildimage/pull/12175
It should not be related to corner case since the build has passed.